game/scripts/npc/abilities/treant_natures_grasp.txt

```diff
@@ -37,7 +37,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage_per_second"                               "30 50 70 90 180 360" //OAA
+        "damage_per_second"                               "30 50 70 90 270 450" //OAA
```

Comment:
I am not a fan of this kind of buffing. May I suggest the scaling like this: 30 60 90 120 240 480